Makefile

```diff
@@ -1,5 +1,11 @@
 SHELL := /bin/bash
-ENABLE_FEATURES ?= default
+ENABLE_FEATURES ?=
+
+ifeq ($(TCMALLOC),1)
+ENABLE_FEATURES += tcmalloc
+else
+ENABLE_FEATURES += jemalloc
```

Comment:
This means we can't choose the system allocator (disabling both `tcmalloc` & `jemalloc`) when using the makefile. Could we change the options to allow this?

Reply:
+1

Reply:
Done.